### PR TITLE
Cooked offal and recipe.

### PR DESCRIPTION
**Pull request by Rivet-the-Zombie**
So we can eat the nasty/nutritious stuff in a cooked format other than
haggis and sausage.

### CODE REVIEW

File: data/json/recipes/recipe_food.json [hunk 1]
```diff
@@ -144,6 +144,32 @@
   ]
 },{
   "type" : "recipe",
+  "result": "offal_cooked",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_MEAT",
+  "skill_used": "cooking",
+  "difficulty": 0,
+  "time": 18000,
+  "reversible": false,
```
Comment (Cyrano7) [inline data/json/recipes/recipe_food.json]: This line can go btw.

File: data/json/recipes/recipe_food.json [hunk 2]
```diff
@@ -144,6 +144,32 @@
...
+  "autolearn": true,
+  "batch_time_factors":[67, 5],
+  "qualities":[
+    {"id":"COOK","level":1,"amount":1}
```
Comment (Cyrano7) [inline data/json/recipes/recipe_food.json]: And this one can be written {"id":"COOK","level":1}

**Comment by Cyrano7** [comment]
This is good for those live in the woods types.

**Comment by Cyrano7** [comment]
@Rivet-the-Zombie I only see the recipe. Forget the comestibles entry?

**Comment by Rivet-the-Zombie** [comment]
Apparently it didn't grab those changes.

File: data/json/items/comestibles/carnivore.json [hunk 3]
```diff
@@ -124,6 +124,20 @@
...
   },
   {
+    "id": "offal_cooked",
+    "copy-from": "meat",
+    "type": "COMESTIBLE",
+    "name": "cooked offal",
+    "name_plural": "cooked offal",
+    "description": "Freshly cooked entrails.  Unappetizing, but filled with essential vitamins.",
+    "relative": {
+        "vitamins": [ [ "vitA", 5 ] ]
```
Comment (mugling) [inline data/json/items/comestibles/carnivore.json]: Did you mean to increase `vitA` here. If so why?
This entry could be condensed to a single line.

Comment (Rivet-the-Zombie) [inline data/json/items/comestibles/carnivore.json]: I did; it's the same vitamin adjustment as raw offal.

Comment (mugling) [inline data/json/items/comestibles/carnivore.json]: Ah right. You should maybe `copy-from` the `offal` item instead?

Comment (Rivet-the-Zombie) [inline data/json/items/comestibles/carnivore.json]: It actually requires a couple less lines to copy from ````meat```` - no need to change calories or parasites etc.

Comment (mugling) [inline data/json/items/comestibles/carnivore.json]: Sure but it's misleading and implies a link between the two where there isn't one?

**Comment by BorkBorkGoesTheCode** [comment]
Will you add pickled animal hooves?

**Comment by Cyrano7** [comment]
@BorkBorkGoesTheCode Let's not get carried away here. Why should that be in the game anyways???

File: data/json/items/comestibles/carnivore.json (hunk 3 above)
Comment (Coolthulhu) [inline data/json/items/comestibles/carnivore.json]: Yeah, copying from offal is a better idea here. If offal gets some significant change, such as vitamins, nutrition, health etc., cooked offal would be adjusted automatically. Say, addition of vitamin D or splitting offal into liver and other offal.

**Comment by Coolthulhu** [comment]
How about limiting offal to boiled or fried? That is, requiring at least a pan (COOK 2) in order to cook it.
I'm having a hard time imagining someone trying to keep a liver from falling apart while only secured by a pointy stick.

**Comment by Rivet-the-Zombie** [comment]
It's not nearly that fragile: [chicken liver skewers](http://www.fauziaskitchenfun.com/sites/default/files/styles/recipe-main/public/wp-content/uploads/2012/09/grilledchickenliver.jpg?itok=beTJkypj).

**Pull request merged**